com.sudovanilla.penpot-deskop.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.sudovanilla.penpot-desktop</id>
+  <launchable type="desktop-id">com.sudovanilla.penpot-desktop.desktop</launchable>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <name>Penpot Desktop</name>
+  <summary>Design Freedom for Teams</summary>
+  <description>
+    <p>
+      Penpot is the first Open Source design and prototyping platform meant for cross-domain teams. Non dependent on operating systems, Penpot is web based and works with open web standards (SVG). For everyone and empowered by the community.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default"><image>https://sudovanilla.com/content/images/penpot-gnome.png</image></screenshot>
+  </screenshots>
+  <categories>
+    <category>Graphics</category>
+  </categories>
+  <url type="homepage">https://penpot.app</url>
```

Comment:
this doesn't look like this is official, so why is it pointing at Penpot?

Reply:
This project is sponsored by Penpot itself, but not official, can be changed to project homepage instead...updating now.

Reply:
is there any communication about that?